upup/models/cloudup/resources/addons/networking.projectcalico.org/k8s-1.16.yaml.template

```diff
@@ -3857,8 +3861,17 @@ spec:
             # The default IPv4 pool to create on startup if none exists. Pod IPs will be
             # chosen from this range. Changing this value after installation will have
             # no effect. This should fall within `--cluster-cidr`.
+            {{- if (WithDefaultBool .Networking.Calico.IPv6Support false) }}
+            - name: CALICO_IPV6POOL_CIDR
+              value: "{{ .KubeControllerManager.ClusterCIDR }}"
+            - name: CALICO_IPV6POOL_NAT_OUTGOING
+              value: "true"
+            - name: CALICO_ROUTER_ID
+              value: "hash"
```

Comment:
Per Calico documentation, this should only be set for IPv6-only configurations. So it should be conditional on `.Networking.Calico.IPv6Support` being `false`.

Reply:
Okay, we're not supporting dual-stack.

Reply:
Exactly. Would have to experiment more with this setting. In theory it should not be needed because would use the host IPv4. In practice, it was needed to get things working. Will be much easier once the `--ipv6` will be available and can add periodic and optional tests.